FluentAssertions.Core/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -101,7 +101,7 @@ public class EquivalencyAssertionOptions : EquivalencyAssertionOptionsBase<Equiv
         public EquivalencyAssertionOptions()
         {
             IncludingNestedObjects();
-            IncludingAllDeclaredProperties();
+            IncludingAllDeclaredMembers();
```

Comment:
I do not disagree with you on the role field should generally have in design.  What is in the balance here is not how users of the library should design their code, but how FA should respond to the code used with it.

I have mentioned before how `ShouldBeEquivalentTo` is one of those features has a "just works" expectation.  If a object does publicly expose a field (even if design wise that is exceptional) it would be expected that it is included.  If that field is not checked then I think false positives are subject to occur in this case because users are used to it working.  If they are lucky they write a test that should fail and have to go though the head-scratching, confusion, and debugging as to what is wrong before finally realizing it is a bug/limitation in FA and not an issue with their code.